### PR TITLE
style: recommend format as an alias in root BUILD

### PR DESCRIPTION
**Pull request by alexeagle**
### Type of change

- Style (white-space, formatting, etc...)

### Test plan

Ran `bazel build :format`

### CODE REVIEW

File: docs/formatting.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ If you use [pre-commit.com](https://pre-commit.com/), add this in your `.pre-com
     - id: bazel-super-formatter
```
Comment (gregmagolan) [inline docs/formatting.md:47]: left over `bazel-super-formatter` ??

Comment (alexeagle) [reply docs/formatting.md:47]: 🍌

File: docs/formatting.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Assuming you installed with the typical layout:
 
 ### Re-format specific file(s)
 
-`bazel run tools:format some/file.md other/file.json`
+`bazel run format some/file.md other/file.json`
```
Comment (gregmagolan) [inline docs/formatting.md:38]: switch this one to `bazel run //:format ...` as well as the one above for consistency?

File: docs/formatting.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ multi_formatter_binary(
 
 Assuming you installed with the typical layout:
 
-`bazel run tools:format`
+`bazel run :format`
```
Comment (gregmagolan) [inline docs/formatting.md:28]: Should mention the alias in the `# installation` block above so the targets in the rest of the doc make sense

**Review by gregmagolan — APPROVED** [review]

File: docs/formatting.md (hunk 3 above)
Comment (alexeagle) [reply docs/formatting.md:28]: yeah I did that right after you read the file :)

**Pull request merged**